### DISCUSSION TITLE
Slight organization of documentation

### DISCUSSION THREAD

**Discussion by in0ni** [MohamedLamineAllal/laravel-mix-glob#69] [Ideas]
Hey, first of all thanks for your time and efforts with this extension. I just want to provide a little feedback regarding your documentation, as I struggled a little 😅.

I would like to suggest your structure as follows, it might help others get what they need quicker:

1. Installation
2. Basic Usage (a couple of simple examples, brief)
3. Reference (this is important to understand the examples) "glb.src() & glb.out() & glb.arg()"
4. How it works
5. Other examples

Maybe it's just me, but I got stuck reading your examples and not understanding what they do, it took me a while to finally get to the section I needed to understand how src/out/arg work.

Just a suggestion, maybe the order isn't perfect but I definitely think: a simple example followed by reference will provide a clear introduction before getting into the various examples.

**Comment by MohamedLamineAllal** [comment]
@in0ni thank you for the feedback. I'm late. But i'll look into structuring it better. And the ref block does clear all better indeed.

@in0ni May i ask. Giving 2023 and 2024. Do you see yourself keep relying on laravel-mix or fully switch to vite ?

**Comment by in0ni** [comment]
Hey @MohamedLamineAllal,

Sounds good, just a minor comment from someone who was unfamiliar with even laravel-mix. To be honest, I was looking to go plain webpack -- but our project lead asked us to use laravel-mix, we are using another framework ;)

**Reply by MohamedLamineAllal** [reply]
Thank you bro. The ecosystem is changing, especially with the new generation of tooling. The latest version of Webpack entirely written in rust (turbopack) is a work in progress. Vite took over the market. Swc replacing Babel...

(Otherwise i have really to do something about GitHub notifications)

**Comment by MohamedLamineAllal** [comment]
Hey @inOni, 
I did update the documentaiton following your advise above. I tried my best. Given my time frame. I shrinked the examples. And I added a better clearing first example. Examples giving the structure and the size of the ref + the last sections. Can't be added at the end. So basically i didn't see a need. The reference already contain examples. 
I did add too a reference with hyperlinks to how it works and the glb.src & glb.out section 
Now it should be easier to pick up. 

You can review it. If you want. Also, you can create a PR with suggestions if you have any.

Thank you a lot.
